src/main/resources/assets/js/components/Marathon.jsx

```diff
@@ -110,10 +172,16 @@ var Marathon = React.createClass({
       error: function () {
         this.setState({fetchState: States.STATE_ERROR});
       }.bind(this),
-      reset: true,
       success: function () {
+        var state = this.state;
```

Comment:
You're getting neither here. This is assigning a reference and now I need to think about whether you've got a local state or are still using `this.state`.